ci-operator/step-registry/osd-ccs/e2e/aws/osd-ccs-e2e-aws-workflow.yaml

```diff
@@ -0,0 +1,24 @@
+workflow:
+  as: osd-ccs-e2e-aws
+  steps:
+    env:
+      COMPUTE_MACHINE_TYPE: "m5.xlarge"
+      MULTI_AZ: "true"
+      COMPUTE_NODES: "3"
+      ETCD_ENCRYPTION: "true"
+      MACHINE_POOL_INSTANCE_TYPE: "m5.2xlarge"
+      ENABLE_AUTOSCALING: "true"
+      MIN_REPLICAS: "3"
+      MAX_REPLICAS: "6"
+    pre:
+    - ref: osd-ccs-osd-ccs-admin-aws
+    - ref: osd-ccs-cluster-provision-aws
+    - ref: osd-ccs-conf-machine-pool
+    - ref: osd-ccs-conf-idp-htpasswd
+    - ref: osd-ccs-cluster-operators-wait-ready
```

Comment:
why not create a single chain for provision cluster step? so the user can just focus on the chain  without care about the changes (detailed steps) in the provision step.